_data/extensions.yaml

```diff
@@ -410,6 +430,13 @@ categories:
         - security
         groupId: io.quarkus
         artifactId: quarkus-elytron-security
+      - name: Elytron Security OAuth 2.0
+        description: "Secure your services using OAuth 2.0"
+        labels:
+        - security
+        - oauth2
+        groupId: io.quarkus
+        artifactId: quarkus-elytron-security-oauth2
```

Comment:
Please add this one at the end of the security section. The other ones should be better choices in most of the cases.